gmso/tests/test_gro.py

```diff
@@ -48,6 +48,10 @@ def test_write_gro(self):
         top = from_parmed(pmd.load_file(get_fn("ethane.gro"), structure=True))
         top.save("out.gro")
 
+    def test_write_gro_with_shift_coord(self):
+        top = from_parmed(pmd.load_file(get_fn("ethane.gro"), structure=True))
+        top.save("out.gro", shift_coord=True)
```

Comment:
read_top = Topology("out.gro")
assert np.all(list(map(lambda x: x.positions>=0, read_top.sites)))

Reply:
Not sure how these comments got resolved, @daico007 did you address them somehow?